src/components/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+import { IconButton } from "src/components/IconButton";
+import { Css, Palette } from "src/Css";
+
+export interface AlertProps {
+  children: ReactNode;
+  onClose: () => void;
+}
+
+export function Alert(props: AlertProps) {
+  const { children, onClose } = props;
+  return (
+    <div css={Css.df.itemsStart.justifyBetween.bgRed100.bRed400.red700.p2.br4.$} role="alert">
```

Comment:
**Suggestion:** Heads up that there is no border added to this component. The `bRed400` adds the colour but I think you need to add the border itself 🙈

Reply:
@KoltonG good catch! I remember typing `ba` and it looked like that didn't work, but I think my webstorm autocomplete was just being funny. Will add!